### PR TITLE
Add a script to upload the GPG key to yum.t.o

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: upload_rpm_gpg_key.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  tasks:
+    - name: "Ensure release directory"
+      file:
+        state: directory
+        path: "{{ releasedir }}/{{ release }}"
+
+    - name: "Ensure GPG key"
+      copy:
+        src: "{{ gpg_key }}"
+        dest: "{{ releasedir }}/{{ release }}/RPM-GPG-KEY-foreman"
```
Comment (evgeni) [inline upload_rpm_gpg_key.yaml:17]:
```suggestion
        dest: "{{ releasedir }}/{{ release }}/RPM-GPG-KEY-{{ project }}"
```

File: upload_yum_gpg [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+fi
+
+upload() {
+	local gpg_key
+	gpg_key=$(mktemp)
+	trap "rm -f $gpg_key" EXIT
+	gpg2 --homedir "$KEYDIR" --armor --export "$FULLGPGKEY" > "$gpg_key"
+
+	copy_unix_pass
+	ansible-playbook --ask-become-pass --inventory=$HOSTS --extra-vars="release=${VERSION} gpg_key=${gpg_key}" upload_rpm_gpg_key.yaml
```
Comment (evgeni) [inline upload_yum_gpg:22]:
```suggestion
	ansible-playbook --ask-become-pass --inventory=$HOSTS --extra-vars="release=${VERSION} gpg_key=${gpg_key} project=${PROJECT}" upload_rpm_gpg_key.yaml
```

**Review by evgeni — CHANGES_REQUESTED** [review]
Yay!

Tiny improvement so it also works for pulpcore :)

**Comment by evgeni** [comment]
Katello uses the Foreman key and so does Client.

**Comment by ekohl** [comment]
But should we special case them to prevent them running or do we just assume people don't?

**Comment by evgeni** [comment]
I'd assume people are smart ;-)

**Review by evgeni — APPROVED** [review]

**Pull request merged**